modules/tools_and_setup.adoc

```diff
@@ -0,0 +1,71 @@
+[id="contributing-to-docs-tools-and-setup"]
+= Install and set up the tools and software
+:icons:
+:linkattrs:
+:description: How to set up and install the tools to contribute
+
+
+== Create a GitHub account
+Before you can contribute to Hybrid Cloud Patterns documentation, you must
+https://www.github.com/join[sign up for a GitHub account].
+
+== Set up authentication
+When you have your account set up, follow the instructions to
+https://help.github.com/articles/generating-ssh-keys/[generate and set up SSH
+keys on GitHub] for proper authentication between your workstation and GitHub.
+
+Confirm authentication is working correctly with the following command:
+
+----
+$ ssh -T git@github.com
+----
+
+== Fork and clone the Hybrid Cloud Patterns documentation repository
+You must fork and set up the Hybrid Cloud Patterns documentation repository on your workstation so that you can create PRs and contribute. These steps must only be performed during initial setup.
+
+. Fork the https://github.com/hybrid-cloud-patterns/docs repository into your
+GitHub account from the GitHub UI. Click *Fork* in the upper right-hand corner.
+
+. In the terminal on your workstation, change into the directory where you want
+to clone the forked repository.
+
+.  Clone the forked repository onto your workstation with the following
+command, replacing _<user_name>_ with your actual GitHub username.
++
+----
+$ git clone git@github.com:<user_name>/docs.git
+----
+
+. Change into the directory for the local repository you just cloned.
++
+----
+$ cd docs
+----
+
+. Add an upstream pointer back to the Hybrid Cloud Patterns's remote repository, in this
+case _docs_.
++
+----
+$ git remote add upstream git@github.com:hybrid-cloud-patterns/docs.git
+----
+
+This ensures that you are tracking the remote repository to keep your local
+repository in sync with it.
+
+== Install Asciidoctor
+The Hybrid Cloud Patterns documentation is created in AsciiDoc language, and is processed with http://asciidoctor.org/[AsciiDoctor], which is an AsciiDoc language processor.
+
+=== Prerequisites
+The following are minimum requirements:
+
+* A bash shell environment (Linux and OS X include a bash shell environment)
+* A web browser (Firefox, Chrome, or Safari)
+* https://docs.asciidoctor.org/asciidoctor/latest/tooling/#web-browser-add-ons-preview-only[Web browser add-ons (preview only)]??
+* An editor that can strip trailing whitespace
+* An https://docs.asciidoctor.org/asciidoctor/latest/install/[installer]
+
+#Install Hugo
+
+#Deploy Hugo as a container image
```

Comment:
Is this a placeholder?

Reply:
Yes, a placeholder for adding instructions later. I have commented it out